packages/snack-sdk/src/transports/CompositedTransport.ts

```diff
@@ -0,0 +1,34 @@
+import TransportImplPubNub from './TransportImplPubNub';
+import TransportImplSocketIO from './TransportImplSocketIO';
+import {
+  SnackTransport,
+  SnackTransportListener,
+  SnackTransportMessage,
+  SnackTransportOptions,
+} from './types';
+
+export default class CompositedTransport implements SnackTransport {
```

Comment:
Generally speaking, I prefer to use the present tense in code (unless the context requires another tense version). It's a personal preference, but maybe worth to do :)

```suggestion
export default class CompositeTransport implements SnackTransport {
```

> **Note**
> This change would also require the file name to change I think